Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -5,16 +5,20 @@ function Add-Tls12InSession {
     param()
 
     try {
-        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
-            [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]3072
+        if ([Net.ServicePointManager]::SecurityProtocol.ToString().Split(',').Trim() -notcontains 'Tls12') {
```

Comment:
this should be tostring().Trim().split(',')

Reply:
Split(',').Trim() - this will trim invidual values. I think this is fine here.

Reply:
Yes, Ajay is right. This should be fine.